pkg/pgmodel/end_to_end_tests/main_test.go

```diff
@@ -102,17 +101,14 @@ func withDB(t testing.TB, DBName string, f func(db *pgxpool.Pool, t testing.TB))
 }
 
 func performMigrate(t testing.TB, DBName string, connectURL string) {
-	dbStd, err := sql.Open("pgx", connectURL)
-	defer func() {
-		err := dbStd.Close()
-		if err != nil {
-			t.Fatal(err)
-		}
-	}()
+	dbStd, err := pgxpool.Connect(context.Background(), connectURL)
```

Comment:
nit: can we change `dbStd` to `migratePool` or something? dbStd was because it wasn't a pgx connection but a sql.DB connection before